content/docs/use-cases/model-registry.md

```diff
@@ -0,0 +1,65 @@
+# Machine Learning Model Registry
+
+A **model registry** is a construct to catalog ML models and their versions.
```

Comment:
sorry, but "construct" is also very unusual ... why not tool?

Reply:
No strong opinion
```suggestion
A **model registry** is a tool to catalog ML models and their versions.
```